source.txt

```diff
@@ -204,7 +204,8 @@ Table of Contents
 
     All folders are treated as existing, and therefore GET requests to
     untouched folders SHOULD be responded to with a folder description
-    with no items (the items field set to '{}'). However, an empty
+    with no items (the items field set to '{}'). ETags of empty folders
+    MUST NOT change value for successive GET requests. However, an empty
     folder MUST NOT be listed as an item in its parent folder.
```

Comment:
I wouldn't mind if a servers uses something depending on the last modified time of a folder as the ETag. That would mean that if a file is created and then deleted in a previously empty folder, subsequent GET requests would get a different ETag. Why would that be such a big problem? Is this just about efficiency?

If we want to add this, then can you change the order of these two sentences, so that the 'However' makes more sense. So:

> All folders are treated as existing, and therefore GET requests to
untouched folders SHOULD be responded to with a folder description
with no items (the items field set to '{}'). However, an empty
folder MUST NOT be listed as an item in its parent folder. [For
(please fill this in) reason], the ETag of an empty folder SHOULD NOT
change value for successive GET requests to the folder's URL.